config/environments/development.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.configure do
+  require 'carrierwave/orm/activerecord'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.